Module-2/newspaper/index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+
+    <!-- Header for the background image, logo, weather icon-->
+    <header>
+        <div class="masthead">
+            <div class="logo">
+            <img src="./assets/images/KnoeLogo.png" width="300" height="100">
+            </div>
+        </div>
+    </header>
+
+    <!-- Navigation bar -->
+    <nav>
+        <div class="nav-items" id = "nav-items">
+            <a href="https://www.knoe.com/news/">News</a>
+            <a href="https://www.knoe.com/weather/">Weather</a>
+            <a href="https://www.knoe.com/sports/">Sports</a>
+            <a href="https://www.knoe.com/page/our-town-directory/">Our Town</a>
+            <a href="https://www.knoe.com/livestream/">Livestream</a>
+            <a href="https://www.knoe.com/contests/">Contests</a>
+        </div>
+    </nav>
+    <main>
+    <!-- This is for the ad space on the left side of the page-->
+    <div class="leftContainer" id="leftContainer">
+        
+    </div>
+
+    <!-- This is for the top middle container between left ad space and 7 day forcast-->
+    <div class="middleContainer" id="middleContainer"> 
+        <div class="top-left-image">
+            <img src="./assets/images/man.png" width="500" height="400">
+        </div>
+      
+    <!-- Middle column with 3 images vertical-->
+        <div class="column">
+            <div class="row">
+                <img src="./assets/images/weathermanImage.png" width="125" height="95">
+                <p>Knoe Thursday Evening Forcast Seasonal Friday, Heating Up Over the Weekend</p>
+            </div>
+            <div class="row">    
+                <img src="./assets/images/womanWithOpenMouth.png" width="125" height="95">
```

Comment:
In future projects, remember to include an alt-tag on any images so Users with disabilities can still use your site